superset/assets/javascripts/dashboard/components/SliceAdder.jsx

```diff
@@ -11,6 +11,8 @@ require('react-bootstrap-table/css/react-bootstrap-table.css');
 const propTypes = {
   dashboard: PropTypes.object.isRequired,
   triggerNode: PropTypes.node.isRequired,
+  user_id: PropTypes.string.isRequired,
```

Comment:
Surprised the linter doesn't complain here...

Reply:
now i renamed all user_id to userId.